doc/other_stuff.md

```diff
@@ -0,0 +1,162 @@
+# Other important stuff
```

Comment:
We definitely need to review the docs, at least to check the styles are rendering properly

/cc @csobier